src/browser-polyfill.js

```diff
@@ -355,18 +378,19 @@ if (typeof browser === "undefined" || Object.getPrototypeOf(browser) !== Object.
        * callback. If the listener function returns a Promise, the response is
        * sent when the promise either resolves or rejects.
        *
-       * @param {*} message
+       * @param {any} message
        *        The message sent by the other end of the channel.
        * @param {object} sender
        *        Details about the sender of the message.
-       * @param {function(*)} sendResponse
+       * @param {function(any):void} sendResponse
        *        A callback which, when called with an arbitrary argument, sends
        *        that value as a response.
+       *
        * @returns {boolean}
        *        True if the wrapped listener returned a Promise, which will later
        *        yield a response. False otherwise.
        */
-      return function onMessage(message, sender, sendResponse) {
+      return function onMessage(message, sender, /** @type {(response?: any) => void} */ sendResponse) {
```

Comment:
The signature of `sendResponse` is already in the jsdoc; remove this duplicate comment.

Reply:
Except that the JSDoc seems to&nbsp;apply to&nbsp;`return`, not&nbsp;`onMessage`

Reply:
Seems more like a bug in the JSDoc parser. Please remove the inline type definition anyway.